op-chain-ops/genesis/helpers.go

```diff
@@ -107,3 +108,18 @@ func newHexBig(in uint64) *hexutil.Big {
 	hb := hexutil.Big(*b)
 	return &hb
 }
+
+func setupEcotone(config *DeployConfig, db *state.MemoryStateDB) (*state.MemoryStateDB, error) {
```

Comment:
The implementation describes a function called `maybeSetupEcotone` since its conditional